cityofaustin/atd-vz-data
# Street/satellite layer toggle doesn't work in Crash location edit map

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/15193

This PR removes the street/satellite layer toggle from the Crash location edit map. It also combines some shared map layers in the maps into a new component.

## Testing
**URL to test:** 
https://deploy-preview-1355--atd-vze-staging.netlify.app/

**Steps to test:**
1. Go to a Crash view and look at the map in the top left corner
2. You should see a token warning on the map and, in your network dev tools, you should see 401 responses from a
`https://api.nearmap.com/tiles` endpoint. These requests for tiles should be happening and should be failing since we can't add a wildcard in URLs added to the allow list for our NearMap token. When this merges, we should see NearMap aerial tiles in all map views in these test steps in staging VZE. (if you are testing locally, we fallback to the Mapbox satellite layer)
3. You should also see street labels when you explore the map.
4. Now, click **Edit Coordinates**, and you should see that the toggle UI for the satellite and street layer is removed. NearMap tile requests should also be returning 401s here and street labels should be present.
5. Last, go to a Location view and look at the map in the top left. Same NearMap 401s and street labels should be present.

---
#### Ship list
